### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by dominikhajduk**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - composer self-update
+
+before_script:
+  - cd taskmanager/
+  - travis_retry composer install
+
+script:
+  - ./vendor/bin/phpcs --standard=PSR2 --encoding=UTF-8 --ignore="*.js" src/ --colors -n -p -s
+  - ./vendor/bin/phpcpd --exclude Tests src/
+  - ./vendor/bin/behat
```
Comment (partikus) [inline .travis.yml:17]: I suggest to change the order, first unit tests, specs and then behat, as an acceptance tests

File: taskmanager/build/logs/pmd.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (partikus) [inline taskmanager/build/logs/pmd.xml:1]: gitignore

File: taskmanager/tests/ExampleTest.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+use App\Application\Task\Task;
+use App\Application\Task\Status;
+use App\Application\User\User;
+
+class ExampleTest extends TestCase
```
Comment (partikus) [inline taskmanager/tests/ExampleTest.php:8]: Do we need this file?

**Comment by dominikhajduk** [comment]
@partikus thanks dude but this PR was just for tests. Anyway I will apply your tips 👍 Sorry for trigger notification.
Keep on track on Wednesday.

**Pull request closed without merging**